addons/document/static/tests/document_tests.js

```diff
@@ -74,5 +78,34 @@ odoo.define('document.tests', function (require) {
             assert.strictEqual(form.sidebar.$('.o_sidebar_delete_attachment').length, 1, "there should be only one attachment");
             form.destroy();
         });
+
+        QUnit.test('no attachment on list view', function (assert) {
+            assert.expect(4);
+
+            var list = createView({
+                View: ListView,
+                model: 'partner',
+                data: this.data,
+                groupBy: ['display_name'],
```

Comment:
do you need it to be grouped?

Reply:
To reproduce before the change, we had to select a record, then trigger a render which keep the selection.

So to test the previous behavior without grouped list view, we would need to call directly in the test:

```js
list.update({}, {keepSelection: true})
```

Reply:
ok then :-)